processor/src/test/java/org/mapstruct/ap/testutil/assertions/JavaFileAssert.java

```diff
@@ -44,15 +44,16 @@ public JavaFileAssert(File actual) {
     /**
      * @return assertion on the file content
      */
-    public StringAssert content() {
+    public AbstractCharSequenceAssert<?, String> content() {
         exists();
         isFile();
 
         try {
             return Assertions.assertThat( Files.toString( actual, Charsets.UTF_8 ) );
         }
         catch ( IOException e ) {
-            failIfCustomMessageIsSet( e );
+            //TODO shouldn't we just fail if something like this happens.
```

Comment:
@gunnarmorling Have a better look at this TODO. `failIfCustomMessageIsSet` does not exists in assertj anymore and I was not sure whether to directly fail with some message or do something else. Previously no custom message was set so even if an exception occurred the test did not fail.